src/util.h

```diff
@@ -31,6 +31,7 @@
 static const bool DEFAULT_LOGTIMEMICROS = false;
 static const bool DEFAULT_LOGIPS        = false;
 static const bool DEFAULT_LOGTIMESTAMPS = true;
+static const int64_t DEFAULT_MAX_TIME_ADJUSTMENT = 70 * 60;
```

Comment:
> then I didn't have to add the include in init.cpp

Yes - that would be slightly better, but everything considered I think it's better to be explicit about dependencies. It's just a symptom of the current centralization of option handling in init.cpp.